boards/arm/adafruit_feather_nrf52840/Kconfig

```diff
@@ -1,10 +0,0 @@
-# Adafruit Feather nRF52840 Express board configuration
-
-# Copyright (c) 2020 Tobias Svehagen
-# SPDX-License-Identifier: Apache-2.0
-
-config BOARD_ENABLE_DCDC
```

Comment:
i.e. restore this

Reply:
I moved this on the advice of @kartben, could you point me to doc describing the differences betwen Kconfig and Kconfig.defconfig?  I didn't see any guidance [here](https://docs.zephyrproject.org/latest/build/kconfig/setting.html#visible-and-invisible-kconfig-symbols) for example

Reply:
`Kconfig.defconfig` is strictly for overriding defaults in zephyr, `Kconfig` is for options that a user can select. @kartben take note

Reply:
FWIW I don't remember suggesting this change so there might have been a misunderstanding :)

Reply:
Will revert